test/token-locking.js

```diff
@@ -0,0 +1,265 @@
+/* globals artifacts */
+
+import { getTokenArgs, checkErrorRevert } from "../helpers/test-helper";
+
+const EtherRouter = artifacts.require("EtherRouter");
+const IColonyNetwork = artifacts.require("IColonyNetwork");
+const IColony = artifacts.require("IColony");
+const ITokenLocking = artifacts.require("ITokenLocking");
+const Token = artifacts.require("Token");
+
+contract("TokenLocking", addresses => {
+  const usersTokens = 6;
+  const userAddress = addresses[1];
+  let token;
+  let tokenLocking;
+  let otherToken;
+  let colonyNetwork;
+  let colony;
+
+  before(async () => {
+    const etherRouter = await EtherRouter.deployed();
+    colonyNetwork = IColonyNetwork.at(etherRouter.address);
+    const tokenLockingAddress = await colonyNetwork.getTokenLocking.call();
+    tokenLocking = ITokenLocking.at(tokenLockingAddress);
+  });
+
+  beforeEach(async () => {
+    let tokenArgs = getTokenArgs();
+    token = await Token.new(...tokenArgs);
+    await token.mint(usersTokens);
+    await token.transfer(userAddress, usersTokens);
+
+    tokenArgs = getTokenArgs();
+    otherToken = await Token.new(...tokenArgs);
+
+    const { logs } = await colonyNetwork.createColony(token.address);
+    const { colonyAddress } = logs[0].args;
+    colony = IColony.at(colonyAddress);
+  });
+
+  describe("when locking tokens", async () => {
+    it("should correctly set colony network address", async () => {
+      await tokenLocking.setColonyNetwork(0x0);
+      let colonyNetworkAddress = await tokenLocking.getColonyNetwork.call();
+      assert.equal(colonyNetworkAddress, 0x0);
+
+      await tokenLocking.setColonyNetwork(colonyNetwork.address);
+      colonyNetworkAddress = await tokenLocking.getColonyNetwork.call();
+      assert.equal(colonyNetworkAddress, colonyNetwork.address);
+    });
+
+    it("should correctly deposit tokens", async () => {
+      await token.approve(tokenLocking.address, usersTokens, {
+        from: userAddress
+      });
+      await tokenLocking.deposit(token.address, usersTokens, {
+        from: userAddress
+      });
+      const userDepositedBalance = await tokenLocking.getUserDepositedBalance.call(token.address, userAddress);
+      assert.equal(userDepositedBalance.toNumber(), usersTokens);
+    });
+
+    it("should not be able todo deposit tokens if they are not approved", async () => {
+      await checkErrorRevert(
```

Comment:
Do you have some editor settings that keeps formatting on multiple lines like this? Our current max len I think is 150 and I don't think you're reaching that.